ansys/mapdl/core/element_commands.py

```diff
@@ -1,23 +1,2796 @@
+
+class _MapdlElementCommands:
+    """MAPDL commands specific to element and meshing operations
+
+    Examples
+    --------
+    Create a single SURF154 element.
+
+    >>> mapdl.prep7()
+    >>> mapdl.et(1, 'SURF154')
+    >>> mapdl.n(1, 0, 0, 0)
+    >>> mapdl.n(2, 1, 0, 0)
+    >>> mapdl.n(3, 1, 1, 0)
+    >>> mapdl.n(4, 0, 1, 0)
+    >>> mapdl.e(1, 2, 3, 4)
+    1
+
+    Create a single hexahedral SOLID185 element
+
+    >>> mapdl.et(2, 'SOLID185')
+    >>> mapdl.type(2)
+    >>> mapdl.n(5, 0, 0, 0)
+    >>> mapdl.n(6, 1, 0, 0)
+    >>> mapdl.n(7, 1, 1, 0)
+    >>> mapdl.n(8, 0, 1, 0)
+    >>> mapdl.n(9, 0, 0, 1)
+    >>> mapdl.n(10, 1, 0, 1)
+    >>> mapdl.n(11, 1, 1, 1)
+    >>> mapdl.n(12, 0, 1, 1)
+    >>> mapdl.e(5, 6, 7, 8, 9, 10, 11, 12)
+    2
+
+    Print the volume of individual elements
+
+    >>> mapdl.clear()
+    >>> output = mapdl.input(examples.vmfiles['vm6'])
+    >>> mapdl.post1()
+    >>> label = 'MYVOLU'
+    >>> mapdl.etable(label, 'VOLU')
+    >>> print(mapdl.pretab(label))
+    PRINT ELEMENT TABLE ITEMS PER ELEMENT
+       *****ANSYS VERIFICATION RUN ONLY*****
+         DO NOT USE RESULTS FOR PRODUCTION
+      ***** POST1 ELEMENT TABLE LISTING *****
+        STAT     CURRENT
+        ELEM     XDISP
+           1  0.59135E-001
+           2  0.59135E-001
+           3  0.59135E-001
+    ...
+
+    See the individual commands for more details.
+
+    """
+
+    def e(self, i: MapdlInt = "", j: MapdlInt = "", k: MapdlInt = "",
+          l: MapdlInt = "", m: MapdlInt = "", n: MapdlInt = "", o:
+          MapdlInt = "", p: MapdlInt = "", **kwargs) -> Optional[int]:
+        """APDL Command: E
+
+        Defines an element by node connectivity.
+
+        Parameters
+        ----------
+        i
+            Number of node assigned to first nodal position (node
+            ``i``).
+
+        j, k, l, m, n, o, p
+            Number assigned to second (node ``j``) through eighth
+            (node ``p``) nodal position, if any.
+
+        Examples
+        --------
+        Create a single SURF154 element.
+
+        >>> mapdl.prep7()
+        >>> mapdl.et(1, 'SURF154')
+        >>> mapdl.n(1, 0, 0, 0)
+        >>> mapdl.n(2, 1, 0, 0)
+        >>> mapdl.n(3, 1, 1, 0)
+        >>> mapdl.n(4, 0, 1, 0)
+        >>> mapdl.e(1, 2, 3, 4)
+        1
+
+        Create a single hexahedral SOLID185 element
+
+        >>> mapdl.et(2, 'SOLID185')
+        >>> mapdl.type(2)
+        >>> mapdl.n(5, 0, 0, 0)
+        >>> mapdl.n(6, 1, 0, 0)
+        >>> mapdl.n(7, 1, 1, 0)
+        >>> mapdl.n(8, 0, 1, 0)
+        >>> mapdl.n(9, 0, 0, 1)
+        >>> mapdl.n(10, 1, 0, 1)
+        >>> mapdl.n(11, 1, 1, 1)
+        >>> mapdl.n(12, 0, 1, 1)
+        >>> mapdl.e(5, 6, 7, 8, 9, 10, 11, 12)
+        2
+
+        Notes
+        -----
+        Defines an element by its nodes and attribute values. Up to 8
+        nodes may be specified with the E command.  If more nodes are
+        needed for the element, use the EMORE command. The number of
```

Comment:
it IS possible, though I need to remember how to do it. I think if we convert to lower case and make it code/monospaced, I _think_ it will "just work". A lot of the docs have been copied from the MAPDL docs so all the unusual aspects are legacy from that probably.

Reply:
This can be done with https://stackoverflow.com/questions/21289806/link-to-class-method-in-python-docstring.  Will require some manual editing